CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/brewing/VanillaBrewingPlus.java

```diff
@@ -0,0 +1,37 @@
+package crafttweaker.mc1120.brewing;
+
+import java.util.List;
+import crafttweaker.api.item.IItemStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Tuple;
+import net.minecraftforge.common.brewing.VanillaBrewingRecipe;
+
+public class VanillaBrewingPlus extends VanillaBrewingRecipe {

```

Comment:
Custom Vanilla Brewing REcipe class

Simply checks if the recipe is removed before pasing the arguments to the actual Function...